exporter/sumologicexporter/config.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+// Config defines configuration for Sumo Logic exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// Global options
+	// Unique URL generated for your HTTP Source.
+	// This is the address to send data to.
+	// url: "https://collectors.sumologic.com/receiver/v1/http/<UniqueHTTPCollectorCode>"
+	URL string `mapstructure:"url"`
+	// Option to enable compression (default true)
+	Compress bool `mapstructure:"compress"`
+	// Compression encoding format, either gzip or deflate (default gzip)
+	CompressEncoding CompressEncodingType `mapstructure:"compress_encoding"`
+	// Max HTTP request body size in bytes before compression (if applied).
+	// By default 1MB is recommended.
+	MaxRequestBodySize int `mapstructure:"max_request_body_size"`
```

Comment:
If this is an http protocol, consider to use one the `confighttp` package for config (see https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/otlphttpexporter/config.go) to have consistency and helper funcs.

Reply:
Done

Reply:
Done